external-builds/pytorch/pytorch_vision_repo.py

```diff
@@ -109,14 +132,11 @@ def add_common(command_parser: argparse.ArgumentParser):
         "save-patches", help="Save local commits as patch files for later application"
     )
     add_common(save_patches_p)
-    save_patches_p.add_argument(
-        "--repo-hashtag",
-        default=default_repo_hashtag,
-        help="Git repository ref/tag to checkout",
-    )
     save_patches_p.set_defaults(func=repo_management.do_save_patches)
 
     args = p.parse_args(cl_args)
+    if args.require_related_commit and not HAS_RELATED_COMMIT:
+        raise ValueError("Could not find torchaudio in pytorch/related-commits")
```

Comment:
Should be torchvision? Also, if it's intended to be the file name, it should be "related_commits" (with an underscore)

Reply:
Yes, I think this should be torchvision. Incorporated your suggestions.